Fix "error: #pragma GCC diagnostic not allowed inside functions"
on older compilers.
--- libupower-glib/up-device.c.orig	2017-09-26 06:53:18.000000000 -0500
+++ libupower-glib/up-device.c	2018-03-27 04:18:09.000000000 -0500
@@ -274,10 +274,7 @@
 	/* get a human readable time */
 	t = (time_t) up_exported_device_get_update_time (priv->proxy_device);
 	time_tm = localtime (&t);
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-y2k"
 	strftime (time_buf, sizeof time_buf, "%c", time_tm);
-#pragma GCC diagnostic pop
 
 	string = g_string_new ("");
 	if (!is_display)
